### PR TITLE
MINT-1710 Revoke, Issue events

### PR DESCRIPTION
**Pull request by e-maximus**

### CODE REVIEW

File: src/components/assetsManager/AddTokenDialog/AddTokenForm.jsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ function mapStateToProps (state) {
 
   return {
     formValues: form.get(FORM_ADD_TOKEN_DIALOG) && form.get(FORM_ADD_TOKEN_DIALOG).get('values'),
-    formErrors: form.get(FORM_ADD_TOKEN_DIALOG) && form.get(FORM_ADD_TOKEN_DIALOG).get('syncErrors'),
+    formErrors: (form.get(FORM_ADD_TOKEN_DIALOG) && form.get(FORM_ADD_TOKEN_DIALOG).get('syncErrors')) || {},
```
Comment (ozalexo) [inline src/components/assetsManager/AddTokenDialog/AddTokenForm.jsx]: Also you may improve mapStateToProps like this:
```
function mapStateToProps (state) {
  const assetsManager = state.get(DUCK_ASSETS_MANAGER)
  const form = state.get('form')

  return {
    formValues: form && form.get('values'),
    formErrors: (form && form.get('syncErrors')) || {},
    platformsList: assetsManager.usersPlatforms(),
  }
}
```

File: packages/core/redux/multisigWallet/actions.js [hunk 2]
```diff
@@ -111,24 +112,24 @@ const subscribeOnWalletManager = () => (dispatch, getState) => {
     })
 
   const wallets = getWallets(getState())
```
Comment (wAISw) [inline packages/core/redux/multisigWallet/actions.js]: remove

File: packages/core/redux/wallets/selectors/balances.js [hunk 3]
```diff
@@ -88,11 +88,12 @@ const balanceCalculator = (walletId, symbol) => createSelector(
   ) => {
     return balances
       .reduce((accumulator, tokenBalance) => {
+        let accumulatorResult = null
         const symbol = Object.keys(tokenBalance)[0]
         const balance = Object.values(tokenBalance)[0]
         const tokenPrice = priceList[symbol] && priceList[symbol][selectedCurrency] || null
-        accumulator += ((balance || 0) * (tokenPrice || 0))
-        return accumulator
+        accumulatorResult = accumulator + ((balance || 0) * (tokenPrice || 0))
```
Comment (wAISw) [inline packages/core/redux/wallets/selectors/balances.js]: add return and remove accumulatorResult

File: packages/login-ui/components/LoginForm/LoginForm.js [hunk 4]
```diff
@@ -72,6 +72,9 @@ function mapDispatchToProps (dispatch) {
     navigateToSelectWallet: () => dispatch(navigateToSelectWallet()),
     initAccountsSignature: () => dispatch(initAccountsSignature()),
     navigateToRecoverAccountPage: () => dispatch(navigateToRecoverAccountPage()),
+    initialValues: {
```
Comment (wAISw) [inline packages/login-ui/components/LoginForm/LoginForm.js]: remove initialValues

File: packages/login-ui/components/LoginWithLedger/LoginWithLedger.js [hunk 5]
```diff
@@ -122,11 +122,11 @@ class LoginLedger extends PureComponent {
       ))
   }
 
-  _buildItem(item, index) {
+  _buildItem (item, index) {
     return <MenuItem value={index} key={index} primaryText={item} />
   }
 
-  handleChange = (event, index, value) => {this.setState({ value }); ledgerProvider.setWallet(this.props.account[index]); networkService.selectAccount(this.props.account[index]);}
+  handleChange = (event, index, value) => {this.setState({ value }); ledgerProvider.setWallet(this.props.account[index]); networkService.selectAccount(this.props.account[index])}
```
Comment (wAISw) [inline packages/login-ui/components/LoginWithLedger/LoginWithLedger.js]: Remove all ;

File: src/components/assetsManager/AddTokenDialog/AddTokenForm.jsx [hunk 6]
```diff
@@ -91,9 +91,16 @@ class Platform extends PureComponent {
   static propTypes = {
     platform: PropTypes.instanceOf(Object),
```
Comment (wAISw) [inline src/components/assetsManager/AddTokenDialog/AddTokenForm.jsx]: Fix propTypes

**Pull request merged**